packages/headless/src/entity/store-branch.ts

```diff
@@ -0,0 +1,34 @@
+export class StoreBranchAddress {
+  id?: string;
+  addressLine1: string;
+  addressLine2?: string;
+  city: string;
+  country: string;
+  postCode: string;
+}
+
+export const enum StoreBranchStatus {
+  PENDING = 'PENDING',
+  ACCEPTED = 'ACCEPTED',
+  REJECTED = 'REJECTED',
+}
+export class StoreBranchInfo {
+  shopName: string;
+  legalName: string;
+}
+
+export class StoreBranchImage {
```

Comment:
From the store branch model, you define the image property as a string. So here is why you define the image property as an object. Remove the StoreBranchImage Object.